gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
-android-gradle-plugin = "8.2.1"
+android-gradle-plugin = "8.2.2"
 androidx-activity-compose = "1.8.2"
 androidx-appcompat = "1.6.1"
-androidx-compose-bom = "2023.10.01"
-androidx-compose-google-fonts = "1.5.4"
-androidx-compose-material3 = "1.2.0-beta02"
-androidx-compose-ui-test-junit4-android = "1.5.4"
+androidx-compose-bom = "2024.01.00"
+androidx-compose-google-fonts = "1.6.0"
+androidx-compose-material3 = "1.2.0-rc01"
+androidx-compose-ui-test-junit4-android = "1.6.0"
```

Comment:
I was about to suggest that you add dependabot for these type of changes but I'm not sure that dependabot has support for the toml gradle dependencies yet.

Reply:
Android Studio automatically checks new versions when you open the file. But sometimes we don't want to update to a particular version. So maybe doing it manually is better...